creator-node/src/routes/prometheusMetricsRoutes.js

```diff
@@ -1,17 +1,36 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
 const express = require('express')
 
 const router = express.Router()
 
 /**
- * Exposes Prometheus metrics at `GET /prometheus_metrics`
+ * Exposes Prometheus metrics for the worker (not aggregated) at `GET /prometheus_metrics_worker`
  */
 
-router.get('/prometheus_metrics', async (req, res) => {
+router.get('/prometheus_metrics_worker', async (req, res) => {
   const prometheusRegistry = req.app.get('serviceRegistry').prometheusRegistry
   const metricData = await prometheusRegistry.getAllMetricData()
 
   res.setHeader('Content-Type', prometheusRegistry.registry.contentType)
   return res.end(metricData)
 })
 
+/**
+ * Exposes Prometheus metrics aggregated across all workers at `GET /prometheus_metrics`
+ */
+router.get('/prometheus_metrics', async (req, res) => {
+  try {
+    const prometheusRegistry = req.app.get('serviceRegistry').prometheusRegistry
+    const metrics = await prometheusRegistry.getCustomAggregateMetricData()
+    res.setHeader(
+      'Content-Type',
+      prometheusRegistry.getCustomAggregateContentType()
+    )
+    res.end(metrics)
```

Comment:
works without it but I'll add the return for consistency

Reply:
i **think** it works without that in this case bc there is no other code to be executed but otherwise it would continue executing
need the return here p sure